Makefile

```diff
@@ -4,7 +4,7 @@ YARN := ./bin/yarn
 
 bootstrap: develop init-config run-dependent-services create-db apply-migrations
 
-develop: ensure-pinned-pip setup-git install-yarn-pkgs install-sentry-dev
+develop: ensure-pinned-pip setup-git install-yarn-pkgs install-dev
```

Comment:
Does getsentry also have a develop target?

Reply:
Yeah, why?

Reply:
Just wanted to double check that there was a consistent 'install everything' make recipie